packages/core/src/components/tabs/tabs.md

```diff
@@ -91,27 +75,18 @@ applications. Here's how you might configure tabs to work with it:
 ```tsx
 import { render } from "react-dom";
 import { Router, Route } from "react-router";
-import { Tabs, TabList, Tab, TabPanel } from "@blueprintjs/core";
+import { Tabs, Tab } from "@blueprintjs/core";
 
 const App = () => { ... };
 
 // keys are necessary in JSX.Element lists to keep React happy
 const contents = [
-    <TabList key={0}>
-        <Tab>Home</Tab>
-        <Tab>Projects</Tab>
-    </TabList>,
-    <TabPanel key={1}>
-        home things
-    </TabPanel>,
-    <TabPanel key={2}>
-        projects things
-    </TabPanel>,
+    <Tab key="home" id="home" title="Home" panel={<HomePanel />}>,
+    <Tab key="proj" id="projects" title="Projects" panel={<ProjectsPanel />}>,
 ];
 
-// using SFCs from TS 1.8, but easy to do without them
-export const Home = () => <Tabs selectedTabIndex={0}>{contents}</Tabs>;
-export const Projects = () => <Tabs selectedTabIndex={1}>{contents}</Tabs>;
+export const Home = () => <Tabs selectedTabId="home">{contents}</Tabs>;
+export const Projects = () => <Tabs selectedTabId="projects">{contents}</Tabs>;
```

Comment:
@adidahiya are these updates correct? is this section still useful?

Reply:
hmm, we should probably remove this react router section because it probably doesn't work with react-router v4. we can add it back if requested by users.